src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -69,7 +69,7 @@ def setUp(self):
             min_version='9999')
 
     def test_all_guids(self):
```

Comment:
It looks like this test case needs to be renamed, like you did below with `test_fetch_blocked_from_db`.

Reply:
https://github.com/mozilla/addons-server/pull/14158/commits/ecdf64d6ae42063feacab214f5f4aff24141818c